unity-renderer/Assets/DCLFeatures/CameraReel/Gallery/Scripts/ThumbnailContextMenuController.cs

```diff
@@ -47,7 +47,7 @@ public class ThumbnailContextMenuController
 
             view.OnShareToTwitterRequested += () =>
             {
-                var description = "Check out what I'm doing in Decentraland right now and join me!";
+                var description = "Check out what I'm doing in Decentraland right now and join me!".Replace(" ", "%20");
```

Comment:
Is there not a more standard way to handle URL encoding to avoid the duplication?